src/Nethermind/Nethermind.Merge.Plugin/Handlers/GetBlobsHandlerV2.cs

```diff
@@ -24,10 +24,13 @@ public Task<ResultWrapper<IEnumerable<BlobAndProofV2>>> HandleAsync(byte[][] req
             return ResultWrapper<IEnumerable<BlobAndProofV2>>.Fail(error, MergeErrorCodes.TooLargeRequest);
         }
 
-        Metrics.NumberOfRequestedBlobs += request.Length;
+        Metrics.GetBlobsRequestsTotal += request.Length;
+
+        var count = txPool.GetBlobCounts(request);
+        Metrics.GetBlobsRequestsInBlobpoolTotal += count;
```

Comment:
Interlocked metrics?

Reply:
We use lots of ++ which are not atomic too. Or you feel like addition is more often to cause issues?

Reply:
`++` are not atomic, we should try fixing places where metrics can be changed from multiple threads and are not synchronized.